molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisWebAppConfig.java

```diff
@@ -415,7 +420,9 @@ protected void reindex()
 		DataServiceImpl localDataService = new DataServiceImpl();
 		EntityManager localEntityManager = new EntityManagerImpl(localDataService);
 		MySqlEntityFactory localMySqlEntityFactory = new MySqlEntityFactory(localEntityManager, localDataService);
-		MetaDataService metaDataService = new MetaDataServiceImpl(localDataService);
+
+		MetaDataServiceImpl metaDataService = new MetaDataServiceImpl(localDataService);
+		metaDataService.setLanguageService(new LanguageService(localDataService, new AppDbSettings()));
```

Comment:
AppDbSettings is spring managed, autowire in this class and pass autowired reference